src/internal/observable/fromEvent.ts

```diff
@@ -20,8 +20,8 @@ export interface NodeCompatibleEventEmitter {
 }
 
 export interface JQueryStyleEventEmitter {
-  on: (eventName: string, handler: Function) => void;
-  off: (eventName: string, handler: Function) => void;
+  on: (eventName: string, handler: NodeEventHandler) => void;
```

Comment:
`NodeEventHandler` appears to be from `@typings/node`. @cartant, am I wrong in saying that this would make us dependent on `@typings/node`?

Reply:
Since that was already used in this file I thought I'd use it again. The important part is that it's `(...args: any[]) => void`, could create a local type for that.

Reply:
> ... this would make us dependent on `@typings/node`?

Nah, this looks okay. The `NodeEventHandler` is declared in this file. I suppose this is okay, but I though jQuery events used a single object parameter - more like the DOM event handlers - rather than multiple parameters. It has been some time since I've used jQuery, so IDK.

Whatever the situation, `Function` is wrong, here, as it's pretty much totally useless, these days, being declared like this:

```ts
interface Function {
    /**
     * Returns the name of the function. Function names are read-only and can not be changed.
     */
    readonly name: string;
}
```